commit b7c2c21d04551a773b81e3ae5ce79fad0aef43e1
Author: Kapil Ropalekar <kapilro@cybage.com>
Date:   Mon Jun 13 20:15:33 2016 +0530

    Fixed warnings under i18n_taxonomy module

diff --git a/i18n_taxonomy/i18n_taxonomy.module b/i18n_taxonomy/i18n_taxonomy.module
index 4053348..cf5984a 100644
--- a/i18n_taxonomy/i18n_taxonomy.module
+++ b/i18n_taxonomy/i18n_taxonomy.module
@@ -458,7 +458,9 @@ function i18n_taxonomy_translate_path($path, $path_prefix = 'taxonomy/term/') {
  */
 function i18n_taxonomy_term_name($term, $langcode = NULL) {
   $key = i18n_object_info('taxonomy_term', 'key');
-  return i18n_taxonomy_vocabulary_mode($term->vid, I18N_MODE_LOCALIZE) ? i18n_string(array('taxonomy', 'term', $term->{$key}, 'name'), $term->name, array('langcode' => $langcode, 'sanitize' => FALSE)) : $term->name;
+  if(isset($term->{$key})){
+    return i18n_taxonomy_vocabulary_mode($term->vid, I18N_MODE_LOCALIZE) ? i18n_string(array('taxonomy', 'term', $term->{$key}, 'name'), $term->name, array('langcode' => $langcode, 'sanitize' => FALSE)) : $term->name;
+  }
 }
 
 
